src/components/search/FacetedFilter/FacetedFilter.tsx

```diff
@@ -0,0 +1,75 @@
+import { graphql } from 'gatsby'
+import React, { useState } from 'react'
+import { useSearch } from 'src/sdk/search/useSearch'
+import type { FacetedFilter_FacetsFragment } from '@generated/FacetedFilter_facets.graphql'
+
+import * as styles from './FacetedFilter.module.css'
+
+interface Props {
+  facets: FacetedFilter_FacetsFragment[]
+}
+
+function FacetedFilter({ facets }: Props) {
+  const [index, setIndex] = useState(-1)
+  const { toggleFacet } = useSearch()
+
+  return (
+    <div className={styles.container}>
+      {facets
+        .filter((facet) => facet.type === 'TEXT')
+        .map(({ name, values }, it) => (
+          <div key={`${name}-${it}`}>
+            <button
+              className={styles.button}
+              onClick={() => setIndex(it)}
+              data-testid="facet-filter-header"
+            >
+              {name}
+            </button>
+            <ul style={{ display: index !== it ? 'none' : 'block' }}>
```

Comment:
Instead of hiding the elements using `display: 'none'`, how about preventing them from rendering? I don't know if it's a good idea to encourage people to hide stuff in the DOM instead of just not putting them there in the first place.